athena-federation-sdk/pom.xml

```diff
@@ -142,6 +142,11 @@
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.9</version>
+        </dependency>
```

Comment:
What are we using from commons-lang3? shoot me a chime message about this since we will need to ensure this plays nice with anything that uses the SDK.

Reply:
Im using this to parse date/datetime